npm/iptm/helper.go

```diff
@@ -0,0 +1,202 @@
+package iptm
+
+import (
+	"fmt"
+
+	"github.com/Azure/azure-container-networking/npm/util"
+)
+
+// GetAllChainsAndRules returns all NPM chains and rules
+func getAllChainsAndRules() [][]string {
+	funcList := []func() [][]string{
+		getAzureNPMChainRules,
+		getAzureNPMIngressPortChainRules,
+		getAzureNPMIngressFromChainRules,
+		getAzureNPMEgressPortChainRules,
+		getAzureNPMEgressToChainRules,
+	}
+
+	chainsAndRules := [][]string{}
+	for _, fn := range funcList {
+		tempRules := fn()
+		chainsAndRules = append(chainsAndRules, tempRules...)
+	}
+
+	return chainsAndRules
+}
+
+// getAzureNPMChainRules returns all rules for AZURE-NPM chain
+func getAzureNPMChainRules() [][]string {
+	// Note: make sure 0th index is prent chain for logging
+	return [][]string{
+		{
+			util.IptablesAzureChain,
```

Comment:
let's make a note to potentially wrap these string slices into structs with validation, the potential for ordering mishaps is pretty high

Reply:
Noted!